cli.js

```diff
@@ -73,6 +77,10 @@ function extensions (arg) {
   return extensions
 }
 
+function stripQuotes (string) {
+  return string.replace(/(^'|")|('|"$)/g, '')
```

Comment:
The regular expression would have to be `/^'|^"|'$|"$/g` to only match quotes in the beginning and the end. I would actually suggest skipping regular expressions all together  and do:

```suggestion
  if (string.startsWith("'") || string.startsWith('"')) string = string.slice(1);
  if (string.endsWith("'") || string.endsWith('"')) string = string.slice(0, -1);
  return string;
```